core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -227,36 +244,61 @@ oppia.controller('FeedbackTab', [
           }
         ]
       }).result.then(function(result) {
-        ThreadDataService.resolveSuggestion(
-          $scope.activeThread.threadId, result.action, result.commitMessage,
-          result.reviewMessage, result.audioUpdateRequired, function() {
-            ThreadDataService.fetchThreads(function() {
-              $scope.setActiveThread($scope.activeThread.threadId);
-            });
-            // Immediately update editor to reflect accepted suggestion.
-            if (result.action === ACTION_ACCEPT_SUGGESTION) {
-              var suggestion = $scope.activeThread.getSuggestion();
-
-              var stateName = suggestion.stateName;
-              var stateDict = ExplorationDataService.data.states[stateName];
-              var state = StateObjectFactory.createFromBackendDict(
-                stateName, stateDict);
-              state.content.setHtml(
-                $scope.activeThread.getReplacementHtmlFromSuggestion());
-              if (result.audioUpdateRequired) {
-                state.contentIdsToAudioTranslations.markAllAudioAsNeedingUpdate(
-                  state.content.getContentId());
+        if (result.action === 'edit'){
+          url = UrlInterpolationService.interpolateUrl(
+            '/suggestionactionhandler/edit/<suggestion_id>', {
+              suggestion_id: $scope.activeThread.suggestion.suggestionId
+            }
+          );// end url object
+          data = {
+            action: result.action,
+            summary_message: result.summaryMessage,
+            change: {
+              property_name: 'content',
+              cmd: 'edit_state_property',
+              old_value: result.oldContent,
+              state_name: result.stateName,
+              new_value: {
+                html: result.newSuggestionHtml
               }
-              ExplorationDataService.data.version += 1;
-              ExplorationStatesService.setState(stateName, state);
-              $rootScope.$broadcast('refreshVersionHistory', {
-                forceRefresh: true
+            }// closing change object
```

Comment:
Do we need these comments? (And the ones below).

Reply:
i used it because its getting confuse which bracket close which function .
If dont't need should i remove?